extension/js/common/api/api.ts

```diff
@@ -194,6 +194,8 @@ export namespace R { // responses
   export type GmailDraftUpdate = {};
   export type GmailDraftGet = { id: string, message: GmailMsg };
   export type GmailDraftSend = {};
+  export type GmailAliases = { sendAs: GmailAlias[] };
+  export type GmailAlias = { sendAsEmail: string, displayName: string, replyToAddress: string, signature: string, isDefault: boolean, treatAsAlias: boolean, verificationStatus: string };
```

Comment:
Looks good but don't export `GmailAlias`, only `GmailAliases`. Actually, I have a convention for this: I recommend to rename `GmailAlias` to `GmailAliases$sendAs`.